cmds/dhclient/dhclient.go

```diff
@@ -247,12 +281,17 @@ func main() {
 			} else {
 				done <- dhclient6(iface, *renewals, timeout, *retry)
 			}
-		}(i)
+			debug("Done dhclient for %v", ifname)
+		}(i.Attrs().Name)
 	}
 
+	go func() {
+		wg.Wait()
```

Comment:
But you'll have to know how many goroutines exactly were launched. That's the only thing.